### PR TITLE
Replace deprecated method with the new one

### PR DESCRIPTION
**Pull request by bennetelli**

### CODE REVIEW

File: backend/src/main/java/com/redhat/cloud/notifications/auth/rbac/RbacIdentityProvider.java [hunk 1]
```diff
@@ -34,6 +34,9 @@
     public static final String RBAC_WRITE_NOTIFICATIONS = "write:notifications";
     public static final String RBAC_READ_INTEGRATIONS_ENDPOINTS = "read:integrations_ep";
     public static final String RBAC_WRITE_INTEGRATIONS_ENDPOINTS = "write:integrations_ep";
+
+    public static String RBAC_READ_ADMIN = "read:admin";
```
Comment (josejulio) [inline backend/src/main/java/com/redhat/cloud/notifications/auth/rbac/RbacIdentityProvider.java:38]: Is this meant to be in this PR ?

Comment (bennetelli) [reply backend/src/main/java/com/redhat/cloud/notifications/auth/rbac/RbacIdentityProvider.java:38]: good catch. No. Just removed it again

File: backend/src/main/java/com/redhat/cloud/notifications/auth/rbac/RbacIdentityProvider.java [hunk 2]
```diff
@@ -34,6 +34,7 @@
     public static final String RBAC_WRITE_NOTIFICATIONS = "write:notifications";
     public static final String RBAC_READ_INTEGRATIONS_ENDPOINTS = "read:integrations_ep";
     public static final String RBAC_WRITE_INTEGRATIONS_ENDPOINTS = "write:integrations_ep";
+
```
Comment (josejulio) [inline backend/src/main/java/com/redhat/cloud/notifications/auth/rbac/RbacIdentityProvider.java:37]: a new line made its way! :D 
```suggestion

```

**Review by josejulio — APPROVED** [review]
lgtm, but a new line made its way to other file

**Pull request merged**